bosh-release/deploy.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
```

Comment:
It seems like deploy-all.sh file has been removed and the same has been added again as a new file with the name deploy.sh. We might need to use git mv to rename this file to be able to preserve commit history.